Jenkinsfile

```diff
@@ -588,7 +588,7 @@ pipeline {
                         }
                     }
                     environment {
-                        GIT_CACHE_PATH = "C:/Users/Administrator/cache"
+                        GIT_CACHE_PATH = "C:\\Users\\Administrator\\cache"
```

Comment:
this is kind of how it was before but we can still try it https://github.com/brave/brave-browser/commit/bcaa25def7a7e8d1b6921475faeb92a9f316502d#diff-58231b16fdee45a03a4ee3cf94a9f2c3L588

Reply:
Do you think `${USERPROFILE}` had double slashes after expanding?